titleMapping.json

```diff
@@ -239,6 +247,106 @@
 		"TocTitle": "Contexts",
 		"PageTitle": "Contexts"
 	},
+	"az datalake": {
+		"TocTitle": "Data Lake",
+		"PageTitle": "Data Lake (Preview)"
+	},
+	"az datalake analytics": {
+		"TocTitle": "Analytics",
+		"PageTitle": "Data Lake Analytics (Preview)"
```

Comment:
Since our service isn't in preview (we're GA), but our commands in CLI 2.0 are, this line may cause confusion.

Is there a better way to label this to avoid this confusion?  Other services may have the same concern (SQL DW, etc.).